src/CLU/Commands.Common.ScenarioTest/ProcessHelper.cs

```diff
@@ -0,0 +1,193 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Text;
+using System.Threading;
+using Microsoft.Azure.Commands.ScenarioTest;
+using Xunit;
+
+namespace Microsoft.Azure.Commands.Examples.Test
```

Comment:
Let's normalize namespaces in this project to something common (like Microsoft.Azure.Commands.ScenarioTest).